pycbc/events/coinc.py

```diff
@@ -612,11 +621,60 @@ def __init__(self, num_templates, analysis_block, background_statistic,
 
         self.singles = {}
 
-        #if save_background_on_interrupt:
-        #    import signal
-        #    def sig_handler(signum, frame):
-        #        pass
-        #    signal.signal(signal.SIGINT, sig_handler)
+    # Adding this here as it requires the results format dict as defined by this
+    # class. 
+    @classmethod
+    def pick_best_coinc(cls, coinc_results):
+        """Choose the best coinc by ifar first, then statistic if needed.
+
+        This function picks which of the available coincs to use. It chooses
```

Comment:
'which.. to use' -> to use for what?  Also might be good to say if this function is used only in multi-ifo time or also in 2-ifo time and say explicitly that each 'coinc' is a 2-ifo coinc.
suggest 'chooses first by ifar ..' -> 'first selects the highest-ifar coinc, then uses highest statistic as a tie-breaker'
trails -> trials
'were possible at this time' -> 'are possible given the active ifos'